components/mpas-ocean/src/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -2149,6 +2197,12 @@ subroutine ocn_time_integration_split_init(domain)!{{{
          useVelocityCorrection = 0.0_RKIND
       endif
 
+      if ( config_vert_advection_method == 'flux-form' ) then
+         configVertAdvMethod = 1
+      elseif ( config_vert_advection_method == 'remap' ) then
+         configVertAdvMethod = 2
+      endif
```

Comment:
Are you thinking that `mpas_ocn_vertical_advection` would have these public parameters and `subroutine ocn_vertical_advection_init` that we can call from `ocn_forward_mode_init`?

Reply:
Yes, that's what I was envisioning.  For now, that module would be super trivial but it would 1) be clear that this one config option and the associated constants belong there and 2) it would be easy to use that module in both forward mode init and diagnostics (and anywhere else it might be needed in the future).